### DISCUSSION TITLE
Theming - type error

### DISCUSSION THREAD

**Discussion by ganeshkrishnasharma** [chakra-ui/chakra-ui#9027] [General] [answered]
I am trying to add my theme, but unfortunately getting a type error.

The error: ( for my custom value `primary` )
```
Type 'string' is not assignable to type 'TokenSchema<PrimitiveTokenValue> | Recursive<TokenSchema<PrimitiveTokenValue>>'.ts(2322)
```

My code:
```
import { createSystem, defineConfig } from "@chakra-ui/react";

const config = defineConfig({
  cssVarsPrefix: "vi",
  theme: {
    tokens: {
      colors: {
        primary: "#EE0F0F",
      },
    },
  },
});

export default createSystem(config);
```

I am using :
"@chakra-ui/react": "^3.0.1"
"typescript": "^5.6.2"
"eslint": "^9.13.0",

Any help to solve this is appreciated. Thanks in advance.

**Comment by segunadebayo** [comment, marked as answer]
The token format changed slightly.

Please see here https://www.chakra-ui.com/docs/theming/tokens
